src/Selenium2Driver.php

```diff
@@ -946,50 +862,68 @@ public function evaluateScript($script)
             $script = 'return ' . $script;
         }
 
-        return $this->wdSession->execute(array('script' => $script, 'args' => array()));
+        return $this->webDriver->executeScript($script);
     }
 
     /**
-     * {@inheritdoc}
+     * Waits some time or until JS condition turns true.
+     *
+     * @param int                                        $timeout   timeout in milliseconds
+     * @param WebDriverExpectedCondition|\Closure|string $condition JS condition / Closure
```

Comment:
I don't think it makes sense to change the documented signature compared to interface

Reply:
returned `inheritdoc`, but the method remains untouched (hidden functionality)